soh/src/overlays/actors/ovl_En_Cow/z_en_cow.c

```diff
@@ -18,6 +18,9 @@ void func_809E0070(Actor* thisx, GlobalContext* globalCtx);
 
 void func_809DF494(EnCow* this, GlobalContext* globalCtx);
 void func_809DF6BC(EnCow* this, GlobalContext* globalCtx);
+struct CowInfo EnCow_GetCowInfo(EnCow* this, GlobalContext* globalCtx);
+void EnCow_MoveCowsForRandomizer(EnCow* this, GlobalContext* globalCtx);
+GetItemID EnCow_GetRandomizerItemFromCow(EnCow* this, GlobalContext* globalCtx);
```

Comment:
nitpick here, and it's probably fine either way, but since these are prefixed with `EnCow`, we probably don't need the extra cow parts in the rest of the names. I'm thinking maybe
```
EnCow_GetInfo
EnCow_MoveForRandomizer
EnCow_GetRandomizedItem
```